docs/install/kata-containers-3.0-rust-runtime-installation-guide.md

```diff
@@ -49,7 +49,7 @@ Follow the [`kata-deploy`](../../tools/packaging/kata-deploy/README.md).
 
 * Download `Rustup` and install  `Rust`
     > **Notes:**
-    > Rust version 1.62.0 is needed
+    > For Rust version, please see versions.yaml file's rust section.
```

Comment:
What I mean is you can do like this
please see [`versions.yaml`](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml) file's rust section.

Reply:
done, tks